README.md

```diff
@@ -143,6 +143,43 @@ def test_something(django_db_blocker):
 
 This does not work when testing migrations, but there is a way: Use `read_fixture` in `cpmonitor/tests/migrations_test.py`.
 
+## Manual tests with data from production
+
+Occasionally, someone with access may provide a copy of the current production database. (See "Server Administration", below.)
+This may be used as follows:
+
+```sh
+# Select the snapshot to use
+SNAPSHOT_NAME=prod_database_<some date found in e2e_tests/database/>
+
+# Remove previous data
+rm db.sqlite3
+rm -r cpmonitor/images/uploads
+
+# Create DB
+python manage.py migrate --settings=config.settings.local
+
+# Optionally migrate back to suitable version. See the corresponding README:
```

Comment:
This line was very confusing for me until I read about the .README files further down, so let's go with something like this
```suggestion
# Optionally migrate back to suitable version (see the .README file corresponding to the snapshot you're about to load):
```